### PR TITLE
Execution

### LINKED ISSUE #11: 5. Execution plan & specifications
•	Execution plan: map based interface

### PR DESCRIPTION
**Pull request by daanschipper**
The section about the execution plan and the specifications.

Resolves #11

### CODE REVIEW

File: Documents/idea document 2/ideadocument2.md [hunk 1]
```diff
@@ -3,20 +3,63 @@
...
+- Real-time input of new taxonomies
+- Use of Face++ to retrieve additional attributes to gain more insight into users
+- Use of Genderize to retrieve additional attributes to gain more insight into users
+- Dynamic selection on additional attributes
+- Display multiple maps next to each other for easy comparison of different filters
+
+### Won't haves
+- Intensity (heat) map
+
+First and foremost the application must be able to visualise the data gathered from Twitter and Strava on a map of Amsterdam. The data, as explained in the Section Methods, is analysed by the BM25 algorithm to rank the relevance to a query. The main priority is to have finish these requirements first so a working prototype can be delivered.
```
Comment (rpjproost) [inline Documents/idea document 2/ideadocument2.md]: first and foremost identify.. not visualize

**Review by rpjproost — CHANGES_REQUESTED** [review]

File: Documents/idea document 2/ideadocument2.md [hunk 2]
```diff
@@ -3,20 +3,63 @@
...
+
+Next the user experience to our platform will be enhanced by giving the user more control over the presentation of the data. The user will be able to view the data based on the time of the day and week.
```
Comment (rpjproost) [inline Documents/idea document 2/ideadocument2.md]: time of the day and week. time of they day and day of week.

**Review by rpjproost — CHANGES_REQUESTED** [review]

File: Documents/idea document 2/ideadocument2.md [hunk 3]
```diff
@@ -3,20 +3,63 @@
...
 With these two sources data which is relevant for the application, can be collected. Data from strava is relevant, because it contains data about athletes accompanied by their locations. Data from Twitter is relevant, since athletes might use Twitter to tell about their sport activities and if that tweet has a geolocation it can be derived where the athlete was conducting its sport. 
-To derive this information after the collection, data must be analyzed, which will be discussed in the Methods section.
+To derive this information after the collection, data must be analyzed, which will be discussed in the Section Methods.
 
 # Methods
 (Thom)
 
-# Speficications & Execution Plan
+# Specifications & Execution Plan
+In the previous sections the objective is presented and the way the data is collected and how this is processed in order to use for the application. In this section the different features as what can be done with this data in the application are described.
```
Comment (rubenstarmans) [inline Documents/idea document 2/ideadocument2.md]: '..the different features as what can be done with this data..' is seen rare zin... Zou dat anders verwoorden

**Review by rpjproost — APPROVED** [review]
I think this can be merged 👍

**Review by rubenstarmans — CHANGES_REQUESTED** [review]

**Review by rubenstarmans — APPROVED** [review]
Dit leest een stuk lekkerder! 👍

**Pull request merged**